Makefile

```
@@ -3,16 +3,17 @@
 ##@ Define the default airbyte-ci version
 AIRBYTE_CI_VERSION ?= latest
 
-# Detect the operating system
+## Detect the operating system
 OS := $(shell uname)
 
 tools.airbyte-ci.install: ## Install airbyte-ci
 	@python airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_install.py ${AIRBYTE_CI_VERSION}
+	@make tools.airbyte-ci.check
```

Instead of calling make from within a make command, can we use the dependency format like

```suggestion
tools.airbyte-ci.install: tools.airbyte-ci.install-binary tools.airbyte-ci.check

tools.airbyte-ci.install-binary: ## Install airbyte-ci
	@python airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_install.py ${AIRBYTE_CI_VERSION}
	@make tools.airbyte-ci.check
```